proof/invariant-abstract/DetSchedAux_AI.thy

```diff
@@ -8,10 +8,9 @@ theory DetSchedAux_AI
 imports DetSchedInvs_AI
 begin
 
-context begin interpretation Arch .
+(* (* FIXME arch_split: check it's global on other arches *)
 requalify_facts
-  invoke_untyped_st_tcb_at
-end
+  invoke_untyped_st_tcb_at *)
```

Comment:
This commented out now, is this getting requalified elsewhere now or was it not used?

Reply:
it was already available in theory context on AARCH64, but I'll need to confirm that's also the case for other arches when I re-deploy there